cts/cts-fencing.in

```diff
@@ -188,6 +203,16 @@ class OutputFoundError(TestError):
         return repr(self.output)
 
 
+class XmlLintError(TestError):
+    """ Exception raised when xmllint fails """
```

Comment:
Since we're leaving open the possibility of other validators in the future, let's make this generic e.g. XmlValidationError